lib/libutils/ext/trace.c

```diff
@@ -15,9 +15,9 @@
 #include <util.h>
 #include <types_ext.h>
 
-#if (TRACE_LEVEL > 0)
+#if (TRACE_LEVEL >= TRACE_MIN)
```

Comment:
If `TRACE_LEVEL` is 0 (`TRACE_MIN`) the below code is now gets built. The behavior before this change was that code was not built. Prefer `#if (TRACE_LEVEL > TRACE_MIN)` or `#if (TRACE_LEVEL >= TRACE_ERROR)`.

Reply:
Correct, I think this behavior should be documented in `mk/config.mk` because it might be a bit counter-intuitive...: if optee_os (more precisely, the user-mode library libutils.a) is built with `CFG_TEE_TA_LOG_LEVEL=0`, TA tracing is disabled regardless of the value of `CFG_TEE_TA_LOG_LEVEL` when the TA is built.

Reply:
I understand your point related to the code guard `#if (TRACE_LEVEL >= TRACE_MIN)`.  I should have stayed consistent with previous implementation. What I did lately aside that fix, was to check at very beginning the trace level:
```
 #if (TRACE_LEVEL < TRACE_MIN) || (TRACE_LEVEL > TRACE_MAX)
 #error "Invalid value of TRACE_LEVEL"
 #endif
```
Once it's within the boundaries, it would include the corresponding code.

@jforissier 
Next was to include a short note in `./mk/config.mk` very closed to your hint comment.